### DISCUSSION TITLE
Propagate add-mask for sensitive values to downstream jobs/runnners

### DISCUSSION THREAD

**Discussion by IceTDrinker** [community/community#11120] [Actions]
Had the case where I masked sensitive values through secrets (some programs would leak parts of other secrets due to how they handled some strings internally, so I added masks for the sub parts they leaked)

But it seems it only worked/per runner/per job while I expected the masking to be consistent for all downstream jobs/runners.

Is it something that is being looked at to avoid repeating the same add-mask commands on all downstream jobs/runners ?

**Comment by ethomson** [comment]
> But it seems it only worked/per runner/per job

That's right - the runner itself is what does the masking.  It would be very challenging in the current architecture to propagate that masking.  I understand the desire, though - we'll give this some thought in the future.  Thanks for the feedback.

**Reply by IceTDrinker** [reply]
While I sympathize with why it may be hard, I think I'm not the only one who has done the mistake and some may not realize masking is maybe not working 🙂 in any case thanks for your answer
